bramkouwen/Music-Mixer
# adding img to images folder

## What does the PR do?
* It adds car-interior.jpg to images folder

## How do you check this PR?
* Pull down the branch
*  See added jpg

## Screenshots 
None.
